distribution/docs/src/main/resources/content/_architecture/_plugins/audit-classified-access-plugin.adoc

```diff
@@ -0,0 +1,24 @@
+:type: plugin
+:status: published
+:title: Audit Classified Access Plugin
+:link: _audit_classified_access_plugin
+:plugintypes: postquery
+:summary: Audits classified access based off specific metacard attributes and configured values.
+
+Audits classified access based off specific metacard attributes and configured values.
```

Comment:
✏️ This sentence is a little vague. "Audits when classified metacard are being accessed. The determination for classification is based on configurable values for the attributes [attribute1], [attribute2]..."